api.py

```diff
@@ -29,6 +29,8 @@
 engine = create_engine(DATABASE_URL)
 SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
 
+EMAILS_ALLOWLIST = os.environ.get("EMAIL_ALLOWLIST", "").split(",")
```

Comment:
@leothomas I think what we want here is to be able to whitelist domains and not just specific email addresses.

cc @kamicut can you confirm that is what the frontend is doing?